cmd/podman/common.go

```diff
@@ -521,6 +521,19 @@ func scrubServer(server string) string {
 	return strings.TrimPrefix(server, "http://")
 }
 
+// HelpTemplate returns the help template for podman commands
+// This uses the short and long options.
+// command should not use this.
+func HelpTemplate() string {
+	return `{{.Short}}
+
+Description:
+
+{{.Long}}
```

Comment:
For consistency with the rest of the output, would you consider (1) removing the empty line between `Description:` and the text, and (2) indenting `{{.Long}}` two spaces?

Reply:
Update: well, that's harder than I thought: multi-line descriptions (import, info, inspect, ...) would need to be manually indented. I see no clean way to deal with this but hope someone can.

Reply:
yeah, I think similar fun here as with the examples.  They'd have to be manually indented.  I think "as is" is probably the best case.

Reply:
I will remove the line for now.